common/app/model/content/Atoms.scala

```diff
@@ -53,6 +54,11 @@ final case class MediaAtom(
       "^0".r.replaceFirstIn(formattedDuration, "") //strip leading zero
     }
   }
+
+  def activeAsset: MediaAsset = activeVersion match {
+    case Some(version) => assets.find(_.version == version).getOrElse(assets.head)
+    case _ => assets.head
+  }
```

Comment:
That's handled in the [calling template](https://github.com/guardian/frontend/blob/master/common/app/views/fragments/atoms/media.scala.html#L7) by showing the poster image instead.

According to the types it's possible to have no assets and no poster image but that would be a genuinely invalid atom so should throw a rendering error I guess?